javascript/list-items.js

```diff
@@ -16,7 +16,7 @@
     },
 
     isThumbsUp: function (x) {
-      return ((x.comment.get('thumbsup') || x.reviewComment.get('approved')) && !x.reviewComment.get('changesRequested'));
+      return ((x.reviewComment.get('approved')) && !x.reviewComment.get('changesRequested'));
```

Comment:
There's an extra set of parentheses here that are no longer needed (around `x.reviewComment.get('approved')`)

Reply:
Good point, squashed them out.